kotlin-react/src/main/kotlin/react/Imports.kt

```diff
@@ -86,3 +86,49 @@ external fun <P : RProps> lazy(loadComponent: () -> Promise<RClass<P>>): RClass<
 // Suspense (16.6+)
 external interface SuspenseProps : RProps
 external val Suspense: RClass<SuspenseProps>
+
+// State Hook
+@JsName("useState")
+external fun <T> rawUseState(initValue: T): RDependenciesArray
+
+// Reducer Hook
+@JsName("useReducer")
+external fun <S, A> rawUseReducer(reducer: RReducer<S, A>, initialState: S): RDependenciesArray
+@JsName("useReducer")
+external fun <S, A> rawUseReducer(reducer: RReducer<S, A>, initialState: S, initialAction: A): RDependenciesArray
```

Comment:
Can we use `initialAction: A = definedExternally` and avoid duplicating the function?

Reply:
I don't sure that it's equal in converted js. And this "optimisation" is unnecessary.